functions/src/Models/Candidate.ts

```diff
@@ -0,0 +1,32 @@
+import firebase from 'firebase-admin';
+import { User } from "./User";
+
+// digest elections model and construct
+export class Candidate {
+	candidateFName: string = "";
+	candidateId: string = "";
+	candidateLName: string = "";
+	candidatePlan: string = "";
+	applyPosition: string = "";
+	approved: boolean = false;
+	votesFromMembers: User[] = []; // mhmmm
```

Comment:
why explain why you have the comment "mhmmm" it's a little confusing just explain why the votesFromMembers field is sus.